sqlcompyre/analysis/table_comparison.py

```diff
@@ -252,12 +252,13 @@ def row_matches(self) -> RowMatches:
     @cached_property
     def column_matches(self) -> ColumnMatches:
         """A comparison between the column values of the two tables."""
+        MATCH_SUFFIX = "_match"
```

Comment:
I think adding an underscore slightly helps to signal that it is internal, but the problem is just shifted if there are two columns `x` and `x__match` in the same table; it's just less likely.

Reply:
There's no need for this to be an expressive name tbh. We could just use something like `zzz_match`?